php/7.1/tests/config--7.1-node10.yaml

```diff
@@ -0,0 +1,10 @@
+schemaVersion: '2.0.0'
+commandTests:
+  - name: "PHP Version"
+    command: "php"
+    args: ["-v"]
+    expectedOutput: ['^PHP 7\.1\.\d+']
+  - name: "Node Version"
+    command: "node"
+    args: ["-v"]
+    expectedOutput: ['^v10\.\d+\.\d+']
```

Comment:
This looks very repetitive. Is there an alternative ?

Reply:
I don't found any - I could use a "base file" that use/load the [node-version]-config.yaml file

Reply:
Maybe it's possible to use ENV there? as so use the matrix information? if we have them.

Reply:
I will merge and open an new Issue explaining our need to simplify this code by using ENV variables if possible